app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_list = command.split(" ")
+    file1 = command_list[1]
+    file2_path = command_list[-1]
+    if "/" in command:
+        directories = command_list[-1].split("/")[:-1]
+        file_path = ""
+        for directory in directories:
+            file_path += f"{directory}/"
```

Comment:
Code Style: Use 'os.path.join' to concatenate parts of the path. This will make your app cross-platform and handle path separators correctly.